pkg/tests/ossm/setup.go

```diff
@@ -101,6 +97,10 @@ func GetSMCPTemplate(version string) string {
 	}
 }
 
+func GetDefaultSMCPTemplate() string {
+	return GetSMCPTemplate(env.GetDefaultSMCPVersion())
+}
+
```

Comment:
Can you move this to its previous position? You effectively just moved this function a few lines down.

Reply:
Was a mistake after a merge from upcoming changes. Ok I'll move it